inlang/source-code/sdk2/src/json-schema/pattern.ts

```diff
@@ -38,12 +38,17 @@ export const Text = Type.Object({
 	value: Type.String(),
 });
 
-export type Declaration = Static<typeof Declaration>;
-export const Declaration = Type.Object({
+export type Input = Static<typeof Input>;
+export const Input = Type.Object({
 	type: Type.Literal("input"),
 	name: Type.String(),
-	value: Expression,
```

Comment:
Removing the possibility for an input to have an expression is the only change at this point.